server/terraform/README.md

```diff
@@ -182,11 +183,13 @@ emacs main.tf
 
 terraform init
 terraform apply
-# Should create all resources without any errors
+# THIS MAY THROW AN ERROR:
```

Comment:
@matthewblain - I'm sure you'll have a comment on this. It is tricky to do this entirely without errors.